orttraining/orttraining/training_ops/cpu/controlflow/common.h

```diff
@@ -1,19 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-#pragma once
-
-namespace onnxruntime {
-namespace contrib {
-
-template <int input_start, int output_start>
-std::vector<std::pair<int, int>> AliasRange(int start, int end) {
-  std::vector<std::pair<int, int>> aliases;
-  for (int i = start; i < end; i++) {
-    aliases.emplace_back(std::pair<int, int>(input_start + i, output_start + i));
-  }
-  return aliases;
-}
```

Comment:
Is this fixed-length approach blocking any features?

Reply:
Robert-XXXL had more then 1024 weights, and hit this issue. 
It's not good to increase the hard coded number either ....